WEB/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-slim
+
+WORKDIR /usr/src/app
+
+ADD ./WEB .
+ADD ./SHARED .
+
+RUN pip3 install -r requirements.txt
+
+CMD ["flask", "run", "--host=0.0.0.0"]
```

Comment:
0.0.0.0 is shorthand for any IP address resulting in the running server being exposed on all of your machine's IP addresses. I can access the app from other computers on my local network after running `docker-compose up`.

Reply:
Hmm, can you still access it after commenting the port 5000:5000 option?

Reply:
I believe if you change it to port 5000:localhost:5000 it should restrict access to localhost

Reply:
It needs to be `127.0.0.1:5000:5000` but if we can set up a build process so that when built locally the port is restricted to `127.0.0.1:5000:5000` and when built on the master branch or on some CI/CD system it's set to `5000:5000` I'd be happy. For now setting it statically to `127.0.0.1:5000:5000` would be fine since we're definitely not putting anything into production soon.

Reply:
FWIW I'd be more than happy to set up that build process tonight.

Reply:
If you want to work on that sure, it’d go well with integrating hot loading, which certainly should not be in the release build. If you want to change that line rn feel free. I’m not able to do it from work.

Reply:
Ok, I've committed the change. I should get back to work myself. 😅